# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/trusted-types/WorkerGlobalScope-eval.html.ini
# Commit: df960f6a3f76
# Full Hash: df960f6a3f76a3fc92fe57054830649c15f4ff3a
# Author: Frédéric Wang <fwang@igalia.com>
# Date: 2025-01-17 21:57:54
# Regressor Bug: 1901492
# File Overlap Count: 1
# Description:
#   Bug 1901492 - Make GetTrustedTypesCompliantString work with workers. r=smaug,tschuster
#   
#   Make sure GetTrustedTypesCompliantString() work when the specified
#   global object is a WorkerGlobalScope. If that happens the function
#   is executed from the worker thread. The following changes are made:
# ==============================================================================

diff -r c6296179fcb4 -r df960f6a3f76 testing/web-platform/meta/trusted-types/WorkerGlobalScope-eval.html.ini
--- a/testing/web-platform/meta/trusted-types/WorkerGlobalScope-eval.html.ini	Fri Jan 17 13:35:40 2025 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,24 +0,0 @@
-[WorkerGlobalScope-eval.html]
-  [eval(string) in dedicated worker]
-    expected: FAIL
-
-  [indirect eval(string) in dedicated worker]
-    expected: FAIL
-
-  [eval(string) with default policy mutation in dedicated worker]
-    expected: FAIL
-
-  [indirect eval(string) with default policy mutation in dedicated worker]
-    expected: FAIL
-
-  [eval(string) in shared worker]
-    expected: FAIL
-
-  [indirect eval(string) in shared worker]
-    expected: FAIL
-
-  [eval(string) with default policy mutation in shared worker]
-    expected: FAIL
-
-  [indirect eval(string) with default policy mutation in shared worker]
-    expected: FAIL